nx-X11/programs/Xserver/hw/nxagent/NXmiexpose.c

```diff
@@ -93,48 +93,9 @@ Equipment Corporation.
 
 /* $Xorg: miexpose.c,v 1.4 2001/02/09 02:05:20 xorgcvs Exp $ */
 
-#ifdef HAVE_DIX_CONFIG_H
-#include <dix-config.h>
-#endif
-
-#include <nx-X11/X.h>
-#define NEED_EVENTS
-#include <nx-X11/Xproto.h>
-#include <nx-X11/Xprotostr.h>
-
-#include "misc.h"
-#include "regionstr.h"
-#include "scrnintstr.h"
-#include "gcstruct.h"
-#include "windowstr.h"
-#include "pixmap.h"
-#include "input.h"
-
-#include "dixstruct.h"
-#include "mi.h"
-#include <nx-X11/Xmd.h>
-
-#include "globals.h"
-
-#ifdef PANORAMIX
-#include "panoramiX.h"
-#include "panoramiXsrv.h"
-#endif
-
-/*
-    machine-independent graphics exposure code.  any device that uses
-the region package can call this.
-*/
-
-#ifdef NXAGENT_SERVER
-
 #include "Windows.h"
 
-#endif
-
-#ifndef RECTLIMIT
-#define RECTLIMIT 25		/* pick a number, any number > 8 */
-#endif
+#include "../../mi/miexpose.c"
 
 /* miHandleExposures 
     generate a region for exposures for areas that were copied from obscured or
```

Comment:
Can also drop `#ifdef NXAGENT_SERVER` checks and complete `#else` branches in this file.